rest_api/controller/documents.py

```diff
@@ -0,0 +1,107 @@
+from datetime import datetime
+from haystack.schema import Document
+from haystack.document_store import sql
+from rest_api.config import LOG_LEVEL
+from typing import Dict
+from fastapi import APIRouter, HTTPException
+import logging
+
+sql = sql.SQLDocumentStore()
```

Comment:
I think we already initialising document store at https://github.com/deepset-ai/haystack/blob/master/rest_api/controller/search.py#L38 , so we can skip initialisation here or we can refactor that piece of code a bit